src/coprocessor/dag/rpn_expr/function.rs

```diff
@@ -1,5 +1,83 @@
 // Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
 
+//! People implementing RPN functions with fixed argument type and count don't necessarily
+//! understand how `Evaluator` and `RpnDef` work. There's a procedure macro called `rpn_fn`
+//! helping you create RPN functions. For example:
+//!
+//! ```
```

Comment:
Will this cause issues when running doc tests?

Reply:
Probably...Let me add the `no_run` attribute.